bigquery/google/cloud/bigquery/schema.py

```diff
@@ -146,6 +167,26 @@ def _key(self):
             self._fields,
         )
 
+    def to_standard_sql(self):
+        """Return the field as the standard SQL field representation object.
+
+        Returns:
+            An instance of :class:`~google.cloud.bigquery_v2.types.StandardSqlField`.
+        """
+        sql_type = types.StandardSqlDataType()
+        sql_type.type_kind = LEGACY_TO_STANDARD_TYPES.get(
+            self.field_type, types.StandardSqlDataType.TYPE_KIND_UNSPECIFIED
+        )
+
+        # NOTE: No need to also handle the "ARRAY" composed type, the latter
+        # does not exist in legacy SQL types.
+        if sql_type.type_kind == types.StandardSqlDataType.STRUCT:  # noqa: E721
```

Comment:
False positive here, `flake8` thinks that is about comparing types and wants to enforce `isinstance()` - but that would not work, as we actually compare integers here.